doc/cygnus-ngsi/installation_and_administration_guide/ngsiv2_support.md

```diff
@@ -0,0 +1,60 @@
+#NGSIv2 support
+Currently, cygnus-ngsi does not support [NGSIv2](http://telefonicaid.github.io/fiware-orion/api/v2/stable/). Only notification format within NGSIv1 is accepted by `NGSIRestHandler`, i.e. something like:
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1290/commits/3c9a517796f3979ee18b9001ae1ff502d000b5f9